src/main/java/appeng/core/sync/packets/PacketJEIRecipe.java

```diff
@@ -224,10 +223,11 @@ public void serverPacketData( final INetworkInfo manager, final AppEngPacket pac
 										whichItem = this.extractItemFromPlayerInventory( player, realForFake, patternItem );
 									}
 
-									craftMatrix.setInventorySlotContents( x, whichItem );
+									craftMatrix.setStackInSlot( x, whichItem );
 								}
 							}
-							con.onCraftMatrixChanged( craftMatrix );
+							// TODO: is this ok? parameter seems to only be used by enchanting table
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)